scripts/ci/generate-circleci-configuration.js

```diff
@@ -81,19 +116,60 @@ const persistToWorkspace = fileName => ( {
 		]
 	};
 
-	config.jobs.cke5_tests_features = {
-		machine: true,
-		steps: [
-			...bootstrapCommands(),
-			prepareCodeCoverageDirectories(),
-			...generateTestSteps( featurePackages, {
-				checkCoverage: true,
-				coverageFile: '.out/combined_features.info'
-			} ),
-			'community_verification_command',
-			persistToWorkspace( 'combined_features.info' )
-		]
-	};
+	// Adding batches to the root `jobs`.
+	featureTestBatches.forEach( ( batch, batchIndex ) => {
+		config.jobs[ featureTestBatchNames[ batchIndex ] ] = {
+			machine: true,
+			steps: [
+				...bootstrapCommands(),
+				prepareCodeCoverageDirectories(),
+				...generateTestSteps( batch, {
+					checkCoverage: true,
+					coverageFile: '.out/combined_features.info'
```

Comment:
This name must be unique per job. Please, do not forget update the coverage job too.

https://github.com/ckeditor/ckeditor5/blob/8b2cd89890ffefc883e900c8cc62ced8a4c34646/.circleci/template.yml#L164